usr/src/uts/armv8/os/mp_startup.c

```diff
@@ -352,11 +352,13 @@ mp_startup_boot(void)
 	/* Let the control CPU continue into tsc_sync_master() */
 	mp_startup_signal(&procset_slave, cp->cpu_id);
 
-	/* Set up the GIC for the new additional CPU */
-	gic_init_secondary(cp->cpu_id);
+	/* Set our trap vector base */
```

Comment:
I think I (and maybe Hayashi too) said "trap vector" in a bunch of places, but I've been trying to say "exception vector" so it matches the ARM